index.html

```diff
@@ -0,0 +1,175 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My First Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap"
+        rel="stylesheet">
+        <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet">
+</head>
+
+<body>
+    <header id="Portfolio" class="header">
+        <h2 class="log"><a class="Welcome" href="#">Welcome</a></h2>
+        <nav class="navbar-humberger">
+            <i class="menuIcon material-icons">menu</i>
+        </nav>
+        <ul class="navbar-links">
+            <li class="close-icon"><i class="closeIcon material-icons d-none">close</i></li>
+            <li class="link"><a href="#Portfolio">Portfolio</a></li>
+            <li class="link"><a href="#About">About</a></li>
+            <li class="link"><a href="#Contact">Contact</a></li>
+        </ul>
+    </header>
+    <main>
+        <section class="section-1">
+            <h1 class="heading-1">Hey there. I’m Josphat</h1>
+            <h2 class="heading-2">I’m a software developer</h2>
+            <p class="par-1"> I can help you build a product , feature or website Look through some of my work and
+                experience! If you like what you see and have a project you need coded,
+                don’t hestiate to
+            </p>
+            <ul class="icons">
+                <li><i class="fa fa-twitter" aria-hidden="true"></i></li>
+                <li><i class="fa fa-linkedin" aria-hidden="true"></i></li>
+                <li><i class="fa fa-medium" aria-hidden="true"></i></li>
+                <li><i class="fa fa-github" aria-hidden="true"></i></li>
+                <li><i class="fa fa-angellist" aria-hidden="true"></i></li>
+            </ul>
+            <div class="down-img">
+                <img src="img/Disabled.png" class="disable" alt="disble">
+            </div>
+        </section>
+
+        <section class="work-section">
+            <div class="section-title">
```

Comment:
![Screenshot from 2022-08-21 03-23-19](https://user-images.githubusercontent.com/93039359/185768080-1e573427-db35-4e2d-b231-d0403306180d.png)

- [ ] You have forgot to add these cards in your portfolio. These cards are actually your projects which have done or you would do in future and also a core requirement of this project so please add them shown in the Figma design and in above image :+1:

Reply:
this was not part of the project requirement. I feel like you are trying to block me from being approved. please check the requirements